### ISSUE TITLE
Cannot get content bubble to show

### ISSUE THREAD

**Issue by ftliou** [OfficeDev/Microsoft-Teams-Samples#370]
Teams dev newbie here, I am trying to get content bubble sample to work but no success so far, here is the repo I am working on: https://github.com/ftliou/ms-teams-sample-content-bubble/tree/master

Basically what I did:
1. Setup Microsoft 365 dev account, with Microsoft 365 E5 Developer license
2. Using teams toolkit on vscode to create new hello bot Teams app, make sure it runs
3. Copy over relevant code from [content bubble sample project](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-content-bubble/nodejs) into my hello bot project
4. For content bubble url, I setup a local server (+ ngrok), the request will return a simple string

What happened after I add the bot to a meeting and start the meeting:
1. During meeting, when I mention bot, I can see agenda items listed in an Adaptive Card
2. After I select any option and click on Push Agenda button, I can only see the feedback form in chat but I cannot see content bubble 

<img width="960" alt="nocontentbubble" src="https://user-images.githubusercontent.com/5211544/174492026-1c7e4316-be01-459c-b6a0-a52b4df75322.png">

log for the activity I sent:
```
{
  type: 'message',
  text: '**Please provide your valuable feedback**',
  inputHint: 'acceptingInput',
  channelData: {
    notification: {
      alertInMeeting: true,
      externalResourceUrl: 'https://teams.microsoft.com/l/bubble/92614d1c-d959-4582-9b03-e0f06155e7f9?url=https%3A%2F%2Fcf32-218-172-11-244.ngrok.io&height=270&width=300&title=ContentBubbleinTeams&completionBotId=92614d1c-d959-4582-9b03-e0f06155e7f9'
    }
  }
}
```
log for returned status after sending content bubble activity:
```
--------- status { id: '1655656883768' }
```

Basically everything works except the content bubble, but sendActivity didn't return error. 
Could someone point me to the right direction?

Thanks in advance!

**Comment by sayali-MSFT** [comment]
@ftliou - We are looking into this I will get back to you soon.

**Comment by sayali-MSFT** [comment]
@ftliou - We are try to repro the scaenario,But not able to repro.It's working fine from our end.
As per document "Meeting content bubble is available only for Azure AD users and not for anonymous users."
Could you please check it once?
Ref Doc-https://docs.microsoft.com/en-us/microsoftteams/platform/sbs-meeting-content-bubble
![image](https://user-images.githubusercontent.com/93527375/174805283-98b26391-10dd-4ebd-8861-f35417bbb524.png)

**Comment by ftliou** [comment]
@sayali-MSFT thanks for the response, nope it's not an anonymous user, the user is admin user from my Microsoft 365 dev account, (with Microsoft 365 E5 Developer license).
And i am also using teams toolkit to do the auto setup, so without going through creating azure bot in azure portal.
Could these setup have made a difference?

**Comment by sayali-MSFT** [comment]
@ftliou -Yes. As you check the document you need to add multiple things in Azure. Could you please tryout with the [sample](https://learn.microsoft.com/en-us/microsoftteams/platform/sbs-meeting-content-bubble).
or provide the console error?
Correct ! The content bubble can only be sent to all meeting participants, i.e cannot be sent to specific target meeting participants

**Comment by garrytrinder** [comment]
> And i am also using teams toolkit to do the auto setup, so without going through creating azure bot in azure portal.

@ftliou I had some issues with the content bubble as well, but was able to get it working in a sample project that uses Teams Toolkit, this might be helpful to you https://github.com/garrytrinder/msteams-meeting-feedback-content-bubble

**Comment by ChetanSharma-msft** [comment]
Hello @ftliou - Sorry for delay in response.
Could you please confirm whether your issue has resolved or need any further help here?

**Comment by ftliou** [comment]
@garrytrinder @ChetanSharma-msft @sayali-MSFT   thanks! I tried again and I was able to get it to work, it turned out I have configured the wrong app id in the url! Didn't see any error message though.
Thanks again.

**Comment by sayali-MSFT** [comment]
@ftliou -We are closing this issue for now, Please feel free to reopen it, if required.

Could you please share your valuable feedback via[ Microsoft Teams Developer Feedback](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR7iCOpS5_b9Nqmwx43u5rtZUOThVR081SllSR05aSDQxQ0tUMDVPTVIxTi4u
) link.
